python/py-libpysal/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-libpysal
+version             4.3.0
+platforms           darwin
+license             BSD
+maintainers         nomaintainers
+
+description         Core components of PySAL A library of spatial analysis functions.
+long_description    ${description}
+
+homepage            https://pysal.org
+
+checksums           rmd160  6872cf4c7f5334003aa05a525ca232da15ef5dad \
+                    sha256  d7cb34aa20f07c93d8f1f12431aa7c6aa9a08b92cc4396022a04fc2086e5d2d7 \
+                    size    2355016
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-pip \
```

Comment:
`py-pip` is not needed, it was only looking for that because you initially missed the `py-pytest-runner` pacakage. Please build in trace-mode with `sudo port -dvt install <portname>`, that will allow you to see which dependencies are actually needed.

Reply:
Thanks done.